php/8.2.dockerfile

```diff
@@ -31,46 +31,35 @@ RUN apt-get update && \
         apt-transport-https\
         libfcgi-bin \
         ca-certificates \
-        curl \
```

Comment:
We should keep curl, as there is a high probability that derived images rely on it